crates/turborepo-lockfiles/src/berry/ser.rs

```diff
@@ -147,27 +147,31 @@ where
     let mut string = String::new();
     let mut first = true;
 
-    let mut add_line = |dependency: &str, field: &str| {
+    let mut add_line = |dependency: &str, field: &str, setting: bool| {
         if !first {
             string.push('\n');
         }
 
         string.push_str(&format!(
-            "    {}:\n      {}: true",
+            "    {}:\n      {}: {}",
             wrap_string(dependency),
-            wrap_string(field)
+            wrap_string(field),
+            setting,
         ));
 
         first = false;
     };
 
     for (dependency, meta) in metadata {
         let dependency = dependency.as_ref();
-        if meta.optional.unwrap_or_default() {
-            add_line(dependency, "optional");
+        if let Some(built) = meta.built {
+            add_line(dependency, "built", built);
         }
-        if meta.unplugged.unwrap_or_default() {
-            add_line(dependency, "unplugged");
+        if let Some(optional) = meta.optional {
+            add_line(dependency, "optional", optional);
+        }
+        if let Some(unplugged) = meta.unplugged {
```

Comment:
After updating this for `built`, I assumed someone may set optional or unplugged to false, and then that too would become an issue.